source/deploying-with-ansible/roles/wazuh-manager.rst

```diff
@@ -43,8 +43,13 @@ Setting the variables in a separate YAML file is recommended when configuring th
       enable: true
       port: 1515
       use_source_ip: 'no'
-      force_insert: 'no'
-      force_time: 0
+      force:
+        - enabled: 'yes'
```

Comment:
I suggest verifying with the CICD team whether this is correct or not. I'm wondering why in the past `force_insert` was set to `no` here when the default value in the configuration was `yes`. Maybe there was a particular reason for that.

Reply:
Documentation was not modified accordingly to the Ansible repository: https://github.com/wazuh/wazuh-ansible/blob/master/roles/wazuh/ansible-wazuh-manager/defaults/main.yml#L368. So, this change looks good to me.